clients/unshaded/src/main/java/tachyon/client/table/AbstractTachyonRawTables.java

```diff
@@ -30,26 +35,34 @@
 @PublicApi
```

Comment:
I would imagine a user to extend the abstract class and override methods as opposed to extending the interface. For what its worth, `AbstractTachyonFileSystem` is also marked PublicApi.